docs/usage.rst

```diff
@@ -12,6 +12,12 @@ Command Line Interface
 
 .. include:: snippets/cli.rst
 
+``--format`` supports the following keywords for replacement: ``filename``,
+``line``, ``definition``, ``message``, ``explanation``, and ``lines``. For
+example:
+
+.. code::
+    %(filename)s:%(line)3d: %(definition)s: %(message)s
```

Comment:
I second @Nurdok's request here. Further, please attempt to render these docs locally. I'm very certain that lines 19 & 20 will not render as you expect. Disregard how GitHub itself renders them because it does the wrong things with reStructuredText 70% of the time.